### ISSUE TITLE
Height of SVG is wrong

### ISSUE THREAD

**Issue by Skagevang** [scratchblocks/scratchblocks#296]
<img width="1128" alt="2019-06-04" src="https://user-images.githubusercontent.com/11090538/58884103-023f2600-86e0-11e9-9d57-6c07b941b6cc.png">

As you can see in the picture the SVG is higher than the actual block. This makes the inline use blocks look bad.

**Comment by tjvr** [comment]
This might be deliberate; I really can't remember. :thinking:

**Comment by NitroCipher** [comment]
Hmm, this is definitely strange, the same issue is _not_ present in s3blocks, so it would have been added after your 3.0 update

**Comment by Skagevang** [comment]
Yeah don't think this is deliberate. Since it's not like this for the scratch2 style.

**Comment by Skagevang** [comment]
What about changing [this line](https://github.com/scratchblocks/scratchblocks/blob/5854607e6e6077199b17f7b270f34504de77582a/scratch3/blocks.js#L750) with
`if (script.blocks.length === 1) height += script.blocks[0].height + 10
else height += script.height`

It's a bit hacky, but seems to work

**Comment by Skagevang** [comment]
And updating the version would be really nice, since #295 isn't in the latest version

**Comment by Skagevang** [comment]
You want me to send a pull request with my suggestion?

**Comment by tjvr** [comment]
Hmm, it might have to do with L747 actually. I'll try and take a look

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Skagevang** [comment]
Hi @tjvr, what's the status for this issue? :)

**Comment by tjvr** [comment]
Sorry, I've been very busy!

I can't see any obvious problems in Document::render(). I wonder if L706 is the culprit...

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Skagevang** [comment]
You are absolutely right @tjvr, the numbers in that line is wrong. It should be:

```
this.height += block.hasPuzzle ? 9 : 1
```

Would be amazing if you fixed this asap and bumped the version  so I can yarn add scratchblocks to the project I'm working on 😄

**Comment by tjvr** [comment]
How do you know those are the right numbers? 😄

If you can PR this, I'll try and take a look. 🙂

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Skagevang** [comment]
This is equal to what I suggested in an earlier comment. And I've tested it.

I can't make a PR, because I already have a fork of another fork of this project. I can't delete that fork before we start to use your repo. And we will start doing that the moment this issue is solved and a new version is released 😄 

It should be just as fast for you to bump version number and change L706 to what I suggested 😉
